tests/console/textinfo.pm

```diff
@@ -8,8 +8,8 @@
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# G-Summary: Rework the tests layout.
-# G-Maintainer: Alberto Planas <aplanas@suse.com>
+# Summary: Rework the tests layout.
+# Maintainer: Alberto Planas <aplanas@suse.com>
```

Comment:
I doubt Alberto wants to maintain this test module anymore. We have to find a better summary and test maintainer. Do you want to add yourself? If not, just add me, `Oliver Kurz <okurz@suse.de>` and provide a useful summary for the test module.

Reply:
Sure, I'll add myself and retype the summary